README.md

```diff
@@ -29,6 +29,12 @@ ALCHEMY_ENDPOINT=
 
 App configuration is located in   `/config/index.js`
 
+## Setting up Mongodb
```

Comment:
If we're lucky enough that dev's are RTM'ing our manual should not tell them to read another manual. Instructions on setting up Mongodb are beyond the scope of our application. 

Also, it works out of the box on default settings so all a dev has to do is install/start it and default config works. Every dev will be different, though - some will use docker, some have system services, etc. But we won't link to those resources in our docs, either.